src/main/java/competition/subsystems/turret/TurretSubsystem.java

```diff
@@ -35,6 +36,12 @@ public TurretSubsystem(CommonLibFactory factory, PropertyFactory pf, IdealElectr
         }
     }
 
+    public void calibrateTurret(){ //here
```

Comment:
I think this whole logic needs to be reworked. You need some set of steps like:
1) Get the "raw" location value coming from the sensor
2) Save that as the offset

Then in `getCurrentAngle()`
1) Read the raw location value
2) Subtract the offset
3) Return that